code/target/src/main/java/com/adobe/marketing/mobile/target/TargetPreviewManager.java

```diff
@@ -388,26 +409,23 @@ private void createAndShowFloatingButton() {
      *
      */
     private void createAndShowMessage() {
-
-        final TargetPreviewFullscreenDelegate previewFullscreenListener = new TargetPreviewFullscreenDelegate(this);
-        final MessageSettings messageSettings = new MessageSettings();
         // Target fullscreen messages are displayed at 100% scale
-        messageSettings.setHeight(FILL_DEVICE_DISPLAY);
-        messageSettings.setWidth(FILL_DEVICE_DISPLAY);
-        messageSettings.setParent(this);
-        messageSettings.setVerticalAlign(MessageSettings.MessageAlignment.TOP);
-        messageSettings.setHorizontalAlign(MessageSettings.MessageAlignment.CENTER);
-        messageSettings.setDisplayAnimation(MessageSettings.MessageAnimation.NONE);
-        messageSettings.setDismissAnimation(MessageSettings.MessageAnimation.NONE);
-        messageSettings.setBackdropColor("#FFFFFF"); // html code for white
-        messageSettings.setBackdropOpacity(1);
-        final FullscreenMessage message = uiService.createFullscreenMessage(webViewHtml, previewFullscreenListener, false , messageSettings);
-
-        if (message != null) {
-            message.show();
-        } else {
+        final InAppMessageSettings inAppMessageSettings = new InAppMessageSettings.Builder()
+                .content(webViewHtml)
+                .verticalAlignment(InAppMessageSettings.MessageAlignment.TOP)
```

Comment:
nit: The `verticalAlignment` refers to the alignment of the presentable wrt the screen. Since the message being constructed here is full screen, explicitly specifying alignment is not necessary. Was there an alignment issue without this?

Reply:
Copied this over from the previous implementation. Removed it and verified that the messages show up fine